packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -902,6 +907,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
     private readonly createContainerMetadata: ICreateContainerMetadata;
     private summaryCount: number | undefined;
+    private readonly sessionExpiryTimeout?: NodeJS.Timeout;
```

Comment:
Please add a comment explaining what this is. Also take a look at the `Timer` utility in "common-utils" for this. It may be useful here. You can find usages in our repo.